pkg/volume/awsebs/attacher.go

```diff
@@ -208,8 +251,13 @@ func (attacher *awsElasticBlockStoreAttacher) MountDevice(spec *volume.Spec, dev
 	notMnt, err := mounter.IsLikelyNotMountPoint(deviceMountPath)
 	if err != nil {
 		if os.IsNotExist(err) {
-			if err := os.MkdirAll(deviceMountPath, 0750); err != nil {
-				return err
+			dir := deviceMountPath
+			if runtime.GOOS == "windows" {
+				// FormatAndMount will mklink here, avoid "Cannot create a file when that file already exists"
+				dir = filepath.Dir(deviceMountPath)
```

Comment:
Im still a bit confused why this is needed, could you add some more details? The same to the later change at `SetUpAt`

Reply:
updated, lemme know if it makes sense now.